tests/test_wallet_master.py

```diff
@@ -352,3 +353,19 @@ def test_transfer_error_safes(
                     asset=asset,
                     from_safe=True,
                 )
+
+    def test_decrypt_mnemonic(
+        self,
+        test_operate: OperateApp,
+    ) -> None:
+        """test_decrypt_mnemonic"""
+        password = test_operate.password
+        wallet_manager = test_operate.wallet_manager
+        mnemonics = create_wallets(wallet_manager)
+
+        assert len(wallet_manager.json) > 0
+        for ledger_type in LedgerType:
+            if wallet_manager.exists(ledger_type):
+                wallet = wallet_manager.load(ledger_type)
+                decrypted_mnemonic = wallet.decrypt_mnemonic(password)
+                assert mnemonics[ledger_type] == decrypted_mnemonic
```

Comment:
let's explicitly test individual `LedgerType`s, so that we ensure that we are getting inside the `if` block

Reply:
Fixed. The test asserts at least there is a wallet.